stdpopsim/cli.py

```diff
@@ -381,7 +381,9 @@ def add_simulate_species_parser(parser, species):
     def run_simulation(args):
         if args.model is None:
             model = stdpopsim.PiecewiseConstantSize(species.population_size)
-            model.citations = species.population_size_citations
+            model.generation_time = species.generation_time
+            model.citations = species.population_size_citations + \
```

Comment:
Safer to ``extend`` this list here. Perhaps we might have a citation in there all ready for the generic model.